cmd/agent/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/gravitational/version"
 
 	log "github.com/sirupsen/logrus"
-	"gopkg.in/alecthomas/kingpin.v2"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
```

Comment:
This is no-op change - the package is already called `kingpin`

Reply:
it was VSCode doing that through `auto-GoFmt` :man_shrugging: I'll revert :+1: